nodejs/average.js

```diff
@@ -1,46 +1,47 @@
-import constants from './constants.js'
-import { Database } from 'sqlite'
-import Redis from 'ioredis'
+import constants from './constants.js';
+import { Database } from 'sqlite';
+import { createClient } from 'redis';
 
-const db = new Database(constants.sqlite_database)
-const redis = new Redis(constants.redis)
+const db = new Database(constants.sqlite_database);
+const client = createClient();
```

Comment:
> @codeDude64 this will always assume Redis is on localhost:6379 with no password. The previous ioredis code used the object `redis` in `constants.js` to specify where Redis is when connecting to it... please change this code here to:
> 
> ```
> const client = createClient({ url: constants.redis });
> ```
> 
> then update `constants.redis` so that the value of `redis` is a Redis URL, for example `redis://localhost:6379` - this will allow people to configure where Redis is and optionally a username and password if needed in one line. See https://github.com/redis/node-redis/blob/master/docs/client-configuration.md for details.

Just to confirm the username and the password are optional in the Redis URL, right?